README.md

```diff
@@ -73,119 +73,143 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- Added a warning about deprecating Video Conference Mute in the future (v0.67), please check https://github.com/microsoft/PowerToys/issues/21473 for more information.
+- The codebase was upgraded to work with .NET 7. Thanks [@snickler](https://github.com/snickler)!
+- Quick Accent can now show a description of the selected character. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+- ColorPicker now supports adding custom formats.
 
 ### Known issues
 
 - The Text Extractor utility [fails to recognize text in some cases on ARM64 devices running Windows 10](https://github.com/microsoft/PowerToys/issues/20278).
 - After installing PowerToys, [the new Windows 11 context menu entries for PowerRename and Image Resizer might not appear before a system restart](https://github.com/microsoft/PowerToys/issues/19124).
 - There are reports of users who are [unable to open the Settings window](https://github.com/microsoft/PowerToys/issues/18015). This is being caused by incompatibilities with some applications (RTSS RivaTuner Statistics Server is a known examples of this). If you're affected by this, please check the  linked issue to verify if any of the presented solutions works for you.
 
+### General
+
+- Downgraded the ModernWPF dependency to 0.9.4 to avoid issues on certain virtualization technologies. (This was a hotfix for 0.64)
+- Upgraded and fixed the code to work with .NET 7. Thanks [@snickler](https://github.com/snickler)!
+
 ### Always on Top
 
-- Detect and put a window on top again if it's no longer on top.
+- Added telemetry for the pinning/unpinning events.
+
+### Awake
+
+- Added telemetry.
+- Don't allow exiting awake from the tray icon when starting from the runner.
 
 ### Color Picker
 
-- Added the hexadecimal integer format. Thanks [@marius-bughiu](https://github.com/marius-bughiu)!
+- Fixed an infinite loop due to a looping UI refresh. (This was a hotfix for 0.64)
+- Added a feature to allow users to create their own color formats.
 
 ### FancyZones
 
-- Added a way for users to configure default layouts for horizontal and vertical screens.
-- Replaced remaining Number Boxes in FancyZones Editor with Sliders, to improve accessibility for screen readers.
-- Fixed an issue breaking window switching shortcuts.
+- Fixed an issue that caused turning off spaces between zones to not apply correctly. (This was a hotfix for 0.64)
+- Prevent the shift key press from trickling down to the focused window. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Fixed a bug causing FancyZones to try resizing hidden windows.
+- Fixed the focus layout preview being empty on first run in the editor.
+- Fixed UI margin in the "Create new layout" dialog.
+- Fixed window positioning issues when switching between virtual desktops.
+- Fixed snapping by hotkey in single zone layouts.
+
+### File explorer add-ons
+
+- Added .log file support to the Monaco preview handler.
```

Comment:
That was a community contribution.

Reply:
Good catch! I've added the community member.